Example/ios/AppAuthExample/Info.plist

```diff
@@ -39,6 +39,11 @@
 	<dict>
 		<key>NSExceptionDomains</key>
 		<dict>
+			<key>demo.identityserver.io</key>
```

Comment:
Is this what need to be done on iOS in order to support insecure connections? If so, should we add a comment to the readme on it?

Reply:
Yeah, could do that. The library already gives you a very informative warning saying the App Transport Security settings needs to be changed to allow HTTP connections. But no harm putting it in the README to save folks some cycles. Will amend 👍